### ISSUE TITLE
Account info is missing delegating amount and returns weird total

### ISSUE THREAD

**Issue by lukaw3d** [oasisprotocol/oasis-indexer#84]
- [ ] There are no fields for how much this account is delegating / debonding (just how much is delegated to it = "escrow")
- [x] Total is weird. Oasisscan's calculation makes more sense
  - (maybe just omit the total and let API consumers figure out what they want for total)

https://index.oasislabs.com/v1/consensus/accounts/oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm :

```js
  "address": "oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm",
  "nonce": 19,
  "available": 7298938395306,
  "escrow": 551969325439709250, // validator specific (delegated to this account)
  "debonding": 5311111413978429, // validator
  "total": 557287735792083000, // = available (account) + escrow (validator) + debonding (validator)
```

https://api.oasisscan.com/mainnet/chain/account/info/oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm :

```js
  "available": "7298.938395306",
  "escrow": "905262.706688650", // bad name
  "debonding": "0",
  "total": "912561.645083956", // = available (account) + escrow (account) + debonding (account)
```

https://monitor.oasis.dev/data/accounts/oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm :

```js
  "liquid_balance": 7298938395306,
  "escrow_balance": 551969325439709250, // validator
  "escrow_debonding_balance": 5311111413978429, // validator
  "delegations_balance": 905262706688650,
  "debonding_delegations_balance": 0,
  "total_balance": 551976624378104600, // weird https://github.com/everstake/oasis-explorer/blob/328dbfa/services/accounts.go#L119

  // oasisindexer total = liquid_balance (account) + escrow_balance (validator) + escrow_debonding_balance (validator) 
  //    = 557287735792083000
  // oasisscan total = liquid_balance (account) + delegations_balance (account) + debonding_delegations_balance (account)
  //    = 912561645083956
```

**Comment by aefhm** [comment]
> (maybe just omit the total and let API consumers figure out what they want for total)

I prefer this the most.

cc @ennsharma

**Comment by aefhm** [comment]
Oops. Sorry, not deployed. 👍

**Comment by aefhm** [comment]
Deployed with `delegations_balance` and `debonding_delegations_balance` referring to self (account) as opposed to "escrow."
```
{
  "address": "oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm",
  "nonce": 19,
  "available": 7509939537999,
  "escrow": 639539901950126300,
  "debonding": 6330914220772295,
  "delegations_balance": 1426579711133057,
  "debonding_delegations_balance": 4337851883403773,
  "allowances": []
}
```
